cli/command/init/espresso.go

```diff
@@ -6,7 +6,18 @@ import (
 	"github.com/saucelabs/saucectl/internal/espresso"
 )
 
+
 func configureEspresso(cfg *initConfig) interface{} {
+	var devices []config.Device
+	var emulators []config.Emulator
+
+	if emulatorIsSet(cfg.emulator) {
+		emulators = append(emulators, cfg.emulator)
+	}
+	if deviceIsSet(cfg.device) {
+		devices = append(devices, cfg.device)
+	}
```

Comment:
Emulator and Device flags track changes just like regular flags do. So you can do:
```suggestion
	if cfg.emulator.Changed {
		emulators = append(emulators, cfg.emulator)
	}
	if cfg.device.Changed {
		devices = append(devices, cfg.device)
	}
```